### ISSUE TITLE
Fonts are loaded over unsecure connection

### ISSUE THREAD

**Issue by robokow** [poole/hyde#82]
in head.html there is the following reference:

```
<link rel="stylesheet" href="http://fonts.googleapis.com/css?family=PT+Sans:400,400italic,700|Abril+Fatface">
```

When website is loaded over secure connection fonts gets loaded over unsecure source

This can easily be solved by:

```
<link rel="stylesheet" href="//fonts.googleapis.com/css?family=PT+Sans:400,400italic,700|Abril+Fatface"> 
```

**Comment by nicolasmccurdy** [comment]
When I look at the usage for fonts on google.com/fonts, they always use `http://` and not `//`. Is there a reason for this?

**Comment by chrylis** [comment]
Actually, the new recommendation is to use _HTTPS_ unconditionally rather than fooling with protocol-relative URLs, which can occasionally interact oddly with non-HTTP pages (such as `file:/`).

**Comment by Tahanima** [comment]
So, I assume adding `https://` should resole the issue?

**Comment by mdo** [comment]
Fix coming in #258.
